apps/site/components/withProgressionSidebar.tsx

```diff
@@ -2,7 +2,11 @@
 
 import ProgressionSidebar from '@node-core/ui-components/Common/ProgressionSidebar';
 import { usePathname } from 'next/navigation';
-import { useTranslations, type RichTranslationValues } from 'next-intl';
+import {
```

Comment:
Can you separate type imports from regular imports?

Reply:
Yup good catch (VSC auto import)